javaparser-core-testing/src/test/java/com/github/javaparser/utils/VisitorSetTest.java

```diff
@@ -138,4 +137,42 @@ void visitorSetToArray() {
         for (CompilationUnit u : set.toArray(new CompilationUnit[2]))
             assertTrue(set.contains(u));
     }
+
+    @Test
+    void containsAllFalse() {
```

Comment:
Please can you keep the method naming convention used in this test class?
Can you rename this method for example visitorSetContainsOnlyPart.